src/filemanager.cpp

```diff
@@ -271,7 +271,7 @@ void FileManager::performWrite()
         exportStream << header[3] << separator << data.size() << "\n";
         exportStream << header[4] << separator << sampleRate << "\n";
         exportStream << header[5] << separator << toolName << "\n";
-	exportStream << header[6] << separator << additionalInformation[0] << "\n";
+	exportStream << header[6] << separator << ((additionalInformation.size() != 0) ? additionalInformation[0] : "") << "\n";
```

Comment:
this looks like it does too much on a single line;
for readability you could use an extra variable;